libpromises/evalfunction.c

```diff
@@ -9673,6 +9700,13 @@ static const FnCallArg GETUID_ARGS[] =
     {NULL, CF_DATA_TYPE_NONE, NULL}
 };
 
+static const FnCallArg USERINGROUP_ARGS[] =
+{
+    {CF_ANYSTRING, CF_DATA_TYPE_STRING, "User name in text"},
+    {CF_ANYSTRING, CF_DATA_TYPE_STRING, "Group name in text"},
```

Comment:
Not sure what you mean by "in text". Do you mean "as string"? Anyways, I think this would be sufficient:
```suggestion
    {CF_ANYSTRING, CF_DATA_TYPE_STRING, "User name"},
    {CF_ANYSTRING, CF_DATA_TYPE_STRING, "Group name"},
```